package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server",
+    "start:secure": "webpack-dev-server --https --inline --hot --content-base dist/ --cert ~/.localhost-ssl/localhost.pem --key ~/.localhost-ssl/localhost.key"
   },
   "author": "Piotr Janik <janikpiotrek@gmail.com>",
```

Comment:
Not relevant to this PR, but I've often wondered whether we (should) have a policy on how to handle these `author` fields. Should they be individual developers or should they always be `developer@concord.org`, for instance?

Reply:
Yeah I'm never sure either. It is kind of nice to have attribution for the people that have worked on the project. But the git history and github repo info also provides that. I guess from my point of view, in most of these cases no one outside of CC will look at this code so it doesn't much matter. But if you want to propose a policy of using developer@concord.org on the `#dev` channel I'd be fine with that.